components/brave_new_tab_ui/apiEventsToStore.ts

```diff
@@ -43,6 +43,10 @@ async function onCustomBackgroundUpdated (customBackground: CustomBackground) {
   getActions().customBackgroundUpdated(customBackground)
 }
 
+async function onSearchPromotionDisabled () {
```

Comment:
Does this need to be async? Currently it doesn't await anything. (you could also just return searchPromotionDisabled(), and remove the async, if it's a promise)

Reply:
Right, changed to just return `searchPromotionDisabled()`